other/docker/compcert/compcert.Dockerfile

```diff
@@ -13,6 +13,7 @@ COPY --from=sources /src/ /work/
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
+RUN git clone --depth=1 https://github.com/jedisct1/libsodium /work/libsodium
```

Comment:
Do we want to pin this to any tag, or is the latest at the time of building always what we want?

Reply:
It's what we want in this build. If something in libsodium master breaks compcert, we want to let them know pretty soon.